pinot-common/src/main/java/org/apache/pinot/common/utils/grpc/GrpcQueryClient.java

```diff
@@ -50,14 +89,52 @@ public GrpcQueryClient(String host, int port, Config config) {
     private static final int DEFAULT_MAX_INBOUND_MESSAGE_BYTES_SIZE = 128 * 1024 * 1024;
     private final int _maxInboundMessageSizeBytes;
     private final boolean _usePlainText;
+    private final String _tlsKeyStorePath;
+    private final String _tlsKeyStorePassword;
+    private final String _tlsKeyStoreType;
+    private final String _tlsTrustStorePath;
+    private final String _tlsTrustStorePassword;
+    private final String _tlsTrustStoreType;
```

Comment:
imo store these properties directly in a `TlsConfig` instance. that way, you can re-use the unified TlsUtils parsing mechanism

Reply:
done.